test/test_new_artifact.sh

```diff
@@ -2,110 +2,27 @@
 
 source test/utils.sh
 
-ARTIFACT_IMAGE_TAG=0.10.1
-
 echo "---------------------------------------------"
-echo "Sending new artifact for mongo"
+echo "- Sending new artifact for mongo             -"
+echo "---------------------------------------------"
+echo ""
 
 # send new artifcat for database
 keptn send event new-artifact --project=$PROJECT --service=carts-db --image=mongo
 verify_test_step $? "Send event new-artifact for carts-db failed"
 
+# lets wait a little bit until we are sure that mongodb has been deployed
+sleep 30
```

Comment:
Why do we need those 30 second sleeps?

Reply:
When the CLI command `keptn send new-artifact` finishes, mongodb should be deployed in dev stage (but not in all the later stages). Therefore I've added this sleep here to make sure it is available in all stages.

Thinking about it now, I am not sure if that's a good idea, because a "real user" or a CI/CD job would also not wait, but just fire the next new-artifact event without taking care of what happened in the past. 
What's your take on that?

Reply:
Got it. 
I would remove this sleep here because `carts-db` is independent of `carts`.
However, the sleep in line 23 is needed.

Reply:
Fixed it.